### ISSUE TITLE
Startup collor selection with problems

### ISSUE THREAD

**Issue by VitorFrost** [darkxst/ihost_indicator#5]
Can we add an option to start with the RGB indicator LED disabled? Also, it shows a black option for color, that doesn't exist in the RGB color wheel. Can this create problems?
Also, the color review are over the color Text
![image](https://github.com/darkxst/ihost_indicator/assets/5491480/9171bd87-a21f-4b20-902d-f7d21dc5e1ca)

**Comment by darkxst** [comment]
Parhaps its not clear, but the toggle switch will disable the RGB LED at startup.
 
 I am using the default colour selector which doesnt have any options
 https://www.home-assistant.io/docs/blueprint/selectors/#rgb-color-selector
 What does the LED do when set to [0,0,0]?
 
 
Not sure why the text is overlapping for you, its ok here.

**Comment by VitorFrost** [comment]
I didn't test it, just to be clear, the ideal solution would be an RGB color wheel just like that. Can't we implement the same selector?
![image](https://github.com/darkxst/ihost_indicator/assets/5491480/25e5646e-9aa4-4f72-a004-c651351f7f54)

**Comment by darkxst** [comment]
yes, that would be nicer, but I didnt find anyway to do that within the configuration yet.

**Comment by darkxst** [comment]
> What does the LED do when set to [0,0,0]?

This just turns the LED off.
